idc_index/index.py

```diff
@@ -177,6 +186,45 @@ def get_idc_version():
         idc_version = Version(idc_index_data.__version__).major
         return f"v{idc_version}"
 
+    def list_indices(self):
+        """
+        Lists all available indices including their installation status.
+
+        Returns:
+            indices_overview (pd.DataFrame): DataFrame containing information per index.
+        """
+
+        return pd.DataFrame.from_dict(self.indices_overview, orient="index")
```

Comment:
What is the motivation to convert to a dataframe here? To me at least, this adds unnecessary complexity.

Reply:
Just to display it in a readable way to the user.

Reply:
I personally would not this function at all. I think it pollutes the API without good reason. I don't think we would want to have a dedicated function that would make a DF for every other dict class variable, right?

If we want a convenience function that would make a DataFrame from a dict, it is better to add a generic helper function that would do this for any dict.